src/silx/gui/colors.py

```diff
@@ -336,7 +336,10 @@ class Colormap(qt.QObject):
     PERCENTILE_1_99 = "percentile_1_99"
     """constant for autoscale using 1st and 99th percentile of data"""
 
-    AUTOSCALE_MODES = (MINMAX, STDDEV3, PERCENTILE_1_99)
+    PERCENTILE_17_5_82_5 = "percentile_17.5_82.5"
+    """constant for autoscale using 17.5 and 82.5 percentile of data"""
+
+    AUTOSCALE_MODES = (MINMAX, STDDEV3, PERCENTILE_1_99, PERCENTILE_17_5_82_5)
```

Comment:
I think the idea was to have a configurable `PERCENTILE`. Your PR was doing that before, or maybe i am wrong.

But with only 3 setup reachable from the GUI.

Or i misunderstand? Maybe double check with @t20100?

Reply:
> I think the idea was to have a configurable PERCENTILE. But with only 3 setup reachable from the GUI.

This is what I understood from our last discussion as well.